lib/features/seller/authentication/view/seller_register_form.dart

```diff
@@ -58,7 +58,7 @@ class SellerRegisterForm extends StatelessWidget {
                     hintText: AppLocalizations.of(context).enter_first_name,
                     labelText: AppLocalizations.of(context).first_name,
                     onTextChange: (value) {
-                      // provider.validateEmail(context, value);
+                      provider.validateName(context, value, true, false);
```

Comment:
In the API i have mentioned the phoneNumber as the mandatory field all other filed are optional.
We can discuss which field we can keep in the Register first step.

Reply:
Now only the phone number will be send to the first step register api for otp registration. Other fields are send as null or empty string.